pynbody/halo/ahf.py

```diff
@@ -363,7 +363,7 @@ def _load_ahf_halos(self, filename):
         f = util.open_(filename,"rt")
         # get all the property names from the first, commented line
         # remove (#)
-        keys = [re.sub('\([0-9]*\)', '', field)
+        keys = [re.sub(r'\([0-9]*\)', '', field)
```

Comment:
Here and in the following, there were many warnings due to invalid escape sequences (e.g. `\(` is an invalid escape sequence). This is fixed by adding the `r` before the string.